plugins/services/src/js/components/modals/EditServiceModal.js

```diff
@@ -5,13 +5,18 @@ import EditFrameworkConfiguration
 import CreateServiceModal
   from "#PLUGINS/services/src/js/components/modals/CreateServiceModal";
 import DCOSStore from "#SRC/js/stores/DCOSStore";
+import FullScreenModal from "#SRC/js/components/modals/FullScreenModal";
 
 class EditServiceModal extends Component {
   render() {
     const { id = "/" } = this.props.params;
     const serviceID = decodeURIComponent(id);
     const service = DCOSStore.serviceTree.findItemById(serviceID);
 
+    if (!service) {
+      return <FullScreenModal open={true} />;
```

Comment:
Hm... now if I go to `http://localhost:4200/#/services/detail/%2Fi-do-not-exist/edit/` it will just render a blank page but I'd rather see an Error message that the service couldn't be found.

Reply:
How bad it would be implementing the message? I'm ok merging 1.11 as is but in 1.12 I'd rather invest in a better user experience.

Reply:
Oh you are right, in that case we have a problem.
I wonder how we can do this without having the error message flash up every time before the endpoint returns the data 🤔 

I would argue that this is quite a scope change from "let's not have this error anymore",would you be ok with me creating a new Jira and pinging design for some UX ideas here?

Reply:
I'm pretty sure the endpoint returns 404 at this point we could make the use of.
But ok, let's create a JIRA

Reply:
Created this Jira: https://jira.mesosphere.com/browse/DCOS_OSS-2355